### DISCUSSION TITLE
History Sync

### DISCUSSION THREAD

**Discussion by berserkwarwolf** [goniszewski/grimoire#38] [Ideas]
**Is your feature request related to a problem? Please describe.**
There's no single self-hosted sync server that takes care of browsinghistory. Except for Firefox Sync server.

**Describe the solution you'd like**
For Grimoire to sync browsing history.

**Describe alternatives you've considered**
Firefox Sync server, which has lots of issues and it's hard to properly set up.

**Comment by goniszewski** [comment]
Hello @berserkwarwolf! An interesting idea, but also one that seems very tricky to implement. I'm curious as to how many people would be willing to take advantage of this feature.

**Comment by berserkwarwolf** [comment]
> Hello @berserkwarwolf! An interesting idea, but also one that seems very tricky to implement. I'm curious as to how many people would be willing to take advantage of this feature.

The majority of people interested in self-hosting their bookmarks will do it because of privacy. Same goes for **every** business in Europe. Without browsing history sync, people won't have a full sync experience on par with Firefox Sync or Google's proprietary alternative. Same goes for passwords, but at least Vaultwarden exists, but there's nothing for browsing history. That's where Grimoire could be one of a kind. The first and only option out there.
